release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2733,6 +2733,47 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+Issued: 2024-07-10
+
+{product-title} release 4.15.21 is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2024:4321[RHSA-2024:4321] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHBA-2024:4324[RHBA-2024:4324] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.21 --pullspecs
+----
+
+[id="ocp-4-15-21-bug-fixes"]
+==== Bug fixes
+
+* Previously, the `alertmanager-trusted-ca-bundle` configmap was not injected into the user-defined Alertmanager container, which prevented the verification of HTTPS web servers receiving alert notifications. With this update, the trusted CA bundle configmap is mounted into the Alertmanager container at the `/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem` path. (link:https://issues.redhat.com/browse/OCPBUGS-36312[*OCPBUGS-36312*])
+
+* Previously, the internal image registry did not correctly authenticate users on clusters configured for `externalAWS` IAM OpenID Connect (OIDC) users. This causes issues for users when pushing or pulling images to and from the internal image registry. With this release, the internal image registry starts by using the `SelfSubjectReview` API instead of the OpenShift-specific user API. The OpenShift-specific user API is not compatible with external OIDC users. (link:https://issues.redhat.com/browse/OCPBUGS-36287[*OCPBUGS-36287*]) 
+
+* Previously, for clusters upgraded from older versions of {product-title}, enabling `kdump` on an OVN-enabled cluster sometimes prevented the node from rejoining the cluster or returning to the `Ready` state. With this release, stale data from older {product-title} versions are removed, so that nodes can now correctly start and rejoin the cluster. (link:https://issues.redhat.com/browse/OCPBUGS-36258[*OCPBUGS-36258*])
+
+* Previously, the {product-title} installation program included a pair of slashes (`//`) in a path to a resource pool for a cluster installed on {vmw-first}. This issue caused the ControlPlaneMachineS (CPMS) Operator to create additional contol plane machines. With this release, the pair of slashes is removed to prevent this issue from occuring. (link:https://issues.redhat.com/browse/OCPBUGS-36225[*OCPBUGS-36225*]) 
+
+// SME check
+* Previously, the GrowPart tool locked a device. This impacted Linux Unified Key Setup-on-disk-format (LUKS) devices from being opened and the operating system booting and and go into emergency mode. With this release, the call to the GrowPart tool is removed, so that LUKS devices are not unintentionally locked and the operating system can successfully boot. (link:https://issues.redhat.com/browse/OCPBUGS-35988[*OCPBUGS-35988*]) 
```

Comment:
Maybe better, actually:
```suggestion
* Previously, the GrowPart tool locked a device. This impacted Linux Unified Key Setup-on-disk-format (LUKS) devices from being opened and caused the operating system to boot into emergency mode. With this release, the call to the GrowPart tool is removed, so that LUKS devices are not unintentionally locked and the operating system can successfully boot. (link:https://issues.redhat.com/browse/OCPBUGS-35988[*OCPBUGS-35988*]) 
```

Reply:
:heavy_check_mark: